spyder/widgets/mixins.py

```diff
@@ -1499,6 +1502,27 @@ def _enter_array(self, inline):
                     self.sig_text_was_inserted.emit()
                 cursor.endEditBlock()
 
+    # ---- Qt methods
+    def mouseDoubleClickEvent(self, event):
+        """select NUMBER tokens to select numeric literals on double click"""
```

Comment:
```suggestion
        """Select NUMBER tokens to select numeric literals on double click."""
```

Minor improvements to this docstring.